src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+const CREATE = 'BOOKSTORE/BOOKS/CREATE_BOOK';
+const DELETE = 'BOOKSTORE/BOOKS/DELETE_BOOK';
+
+// Reducer
+export default function reducerBooks(state = [], action = {}) {
+  switch (action.type) {
+    case CREATE:
+      return state.push(action.payload);
+    case DELETE:
+      return state.filter((book) => (book.id !== action.payload));
+    default: return state;
+  }
+}
+
+// Action Creators
+export function createBook(book) {
+  return { type: CREATE, payload: book };
+}
+
+export function deleteBook(id) {
+  return { type: DELETE, payload: id };
+}
```

Comment:
- [ ] Please users find it difficult to `Add New Book` to the store, please try to fix it. 
- [ ] Please users find it difficult to `Remove Book` from the store, please try to fix it too.

Reply:
@CollinsTatang

 **can you please explain more what the required changes? what do you mean by ' Please users find it difficult ' ??**
I think you have a typo or smthg

Reply:
Ok
Sorry about the error 
let me reach out for correction

Reply:
Hi @ichala 
I am very sorry the problem will be addressed soon.